### PR TITLE
add make-sense

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @ndmaxar agree to be maintainers?

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/make-sense/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  url: https://github.com/SkalskiP/make-sense/archive/{{ version }}-alpha.tar.gz
+  sha256: c61b2a5926c93da57694a42f29efd14cb99f1fb313be76269babe6efb78bbecb
+
+build:
+  number: 0
+  script:
+    - tgz=$(npm pack)  # [not win]
+    - npm install -g $tgz  # [not win]
+    - FOR /F "delims=" %%i IN ('npm pack') DO set "tgz=%%i"  # [win]
+    - npm install -g %tgz%  # [win]
```
Comment (scopatz) [inline recipes/make-sense/meta.yaml:18]: These should be in build.sh and bld.bat scripts.

File: recipes/make-sense/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (scopatz) [inline recipes/make-sense/meta.yaml:22]: Do you need a C++ compiler for an npm package?

Comment (rluria14) [reply recipes/make-sense/meta.yaml:17]: yes, i tried building without and kept getting an error that required the compiler

Comment (scopatz) [reply recipes/make-sense/meta.yaml:17]: Weird.... CC @isuruf @beckermr

**Comment by rluria14** [comment]
@conda-forge-admin please restart ci

File: recipes/make-sense/meta.yaml (hunk 2 above)
Comment (chrisburr) [reply recipes/make-sense/meta.yaml:17]: It's compiling libsass which isn't installed itself but is used for generating the installed CSS files

File: recipes/make-sense/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SkalskiP/make-sense/archive/{{ version }}-alpha.tar.gz
+  sha256: c61b2a5926c93da57694a42f29efd14cb99f1fb313be76269babe6efb78bbecb
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/make-sense/meta.yaml:13]: I guess the built package doesn't have any platform dependence?
```suggestion
  number: 0
  noarch: generic
```

Comment (rluria14) [reply recipes/make-sense/meta.yaml:13]: thanks, just added that

Comment (rluria14) [reply recipes/make-sense/meta.yaml:13]: @chrisburr getting an overlinking error on linux with noarch

**Comment by rluria14** [comment]
@conda-forge/staged-recipes please review

**Comment by rluria14** [comment]
@chrisburr @scopatz @conda-forge/staged-recipes ping! please review, thanks

File: recipes/make-sense/meta.yaml (hunk 3 above)
Comment (chrisburr) [reply recipes/make-sense/meta.yaml:13]: Ah yes, the weird compiler dependence means you need:
```suggestion
  number: 0
  noarch: generic
  ignore_run_exports:
    - libstdc++
```

Comment (rluria14) [reply recipes/make-sense/meta.yaml:13]: @chrisburr still got the same error. tried `libstdcxx-ng` and `libgcc-ng` in ignore_run_exports as well but those did also not work

Comment (rluria14) [reply recipes/make-sense/meta.yaml:13]: @chrisburr seems to be fixed now with missing_dso_whitelist for `libgcc_s.so.1` and `libstdc++.so.6`

**Pull request merged**